### DISCUSSION TITLE
H115i Linux Mint system D

### DISCUSSION THREAD

**Discussion by galtline** [liquidctl/liquidctl#593] [General]
I have the following service file loading on start up. My question is it normal to have the fans increase in speed with every action on the computer. Every time I open a web browser the fans increase for a few seconds and then resume. In addition it seems that my computer Mint 21.1 runs very sluggish and laggy.  I have a fresh install via USB of Linux Mint 21.1 so I am not sure what is going on. 

[Unit]
Description=AIO startup service

[Service]
Type=oneshot
ExecStart=liquidctl initialize
ExecStart=liquidctl --match H115i set fan speed  20 40  30 60  34 90  40 90  50 100

[Install]
WantedBy=default.target

**Comment by aleksamagicka** [comment]
Perhaps look at the CPU temps? Is the pump working at all?

**Reply by galtline** [reply]
Can you control the pump speed on this unit

**Reply by aleksamagicka** [reply]
I see at least five H115i variants in the README and you haven't said which one you have, so I don't know. You can try setting the pump speed, see if it gives you an error. But if the whole system is laggy, that may be indicative of the CPU throttling. Watch over CPU and liquid temps to see if anything looks out of order.

**Comment by galtline** [comment]
CORSAIR iCUE H150i RGB PRO XT

**Reply by galtline** [reply]
I removed the config file from system d and uninstalled the liquid from linx and now my completer works fine. The software is causing my computer to jam up and become very laggy.

**Comment by galtline** [comment]
I updated to the newest version as the version coming from software is version 1.5.1-1 . Looks like its working
